voip_oca/static/src/components/softphone/softphone.scss

```diff
@@ -2,7 +2,7 @@
     width: 300px;
     height: auto;
     max-height: 70%;
-    z-index: 99;
+    z-index: 1001;
```

Comment:
I would say this z-index is on purpose for not being on top on certain cases, isn't it @etobella ?

Reply:
Ping @etobella